docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -23,16 +23,12 @@ As session data of the external {jdgserver_name} is also cached in the Infinispa
 
 In the following paragraphs and diagrams, references to deploying {jdgserver_name} apply to the external {jdgserver_name}.
 
-image::high-availability/active-passive-sync.dio.svg[]
+image::high-availability/active-active-sync.dio.svg[]
 
 == Causes of data and service loss
 
 While this setup aims for high availability, the following situations can still lead to service or data loss:
 
-* Network failures between the sites or failures of components can lead to short service downtimes while those failures are detected.
-The service will be restored automatically.
-The system is degraded until the failures are detected and the backup cluster is promoted to service requests.
-
```

Comment:
You are right that this needs some re-wording, still I think it shouldn't be removed altogether. 
When one site fails, some requests that are routed to that site might still be lost, although the failover should now be a lot faster in the new setup.